docs/changehistory/NextVersion.md

```diff
@@ -29,3 +29,18 @@ The synchronous `void`-returning overload of [IModelTransformer.initFromExternal
 It will still perform the old behavior synchronously until it is removed. It will now however return a `Promise` (which should be
 awaited) if invoked with the an [InitFromExternalSourceAspectsArgs]($transformer) argument, which is necessary when processing
 changes instead of the full source contents.
+
+## AppUI
```

Comment:
i would make a separate appui 4.0 next ver md for now

Reply:
This is mostly a placeholder until we figure how this is going to be, I dont mind creating a different file if that's better.